lib/WP_Auth0_LoginManager.php

```diff
@@ -551,7 +561,7 @@ protected function get_state( $decoded = FALSE ) {
 
     if ( empty( $this->state ) ) {
       // Get and store base64 encoded state
-      $state_val = $this->query_vars( 'state' );
+      $state_val = $_REQUEST[ 'state' ];
```

Comment:
Why, did the previous stop working?

Reply:
`$this->query_vars()` is not explicit enough, makes for confusion around how these vars are handled. I want `state` from either POST or GET (aka `$_REQUEST`), I don't want anything else.